example/ios/AdyenExample.xcodeproj/project.pbxproj

```diff
@@ -474,7 +466,7 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = AdyenExampleTests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.6;
```

Comment:
Does new RN architecture require this bump?

Reply:
This is default minimum requirement and React Native `0.77.1` - it sets 15.6 via Cocoapods under the hood